### DISCUSSION TITLE
Replacing CC2531 with CC2531 guide

### DISCUSSION THREAD

**Discussion by rbswift** [Koenkk/zigbee2mqtt#8702] [General]
Hi, I have purchased a CC2531 with antenna to replace my CC2531 with onboard antenna. Can I just swap them over or do I need to follow a migration process? I would prefer not to have to join all my devices again. Thanks in advance

**Comment by Koenkk** [comment]
Yes, just swap them, no repairing needed (see https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices)

**Comment by rbswift** [comment]
Great thanks. I saw that FAQ but it didn't seem to cover this case so wanted to check. Cheers
